src/apify_client/clients/resource_clients/actor.py

```diff
@@ -0,0 +1,310 @@
+
+    def get(self) -> Optional[Dict]:
+        """Retrieve the actor.
+
+        https://docs.apify.com/api/v2#/reference/actors/actor-object/get-actor
+
+        Returns:
+            dict, optional: The retrieved actor
+        """
+        return self._get()
+
+    def update(
+        self,
+        *,
+        name: Optional[str] = None,
+        title: Optional[str] = None,
+        description: Optional[str] = None,
+        seo_title: Optional[str] = None,
+        seo_description: Optional[str] = None,
+        versions: Optional[List[Dict]] = None,
+        restart_on_error: Optional[bool] = None,
+        is_public: Optional[bool] = None,
+        is_deprecated: Optional[bool] = None,
+        is_anonymously_runnable: Optional[bool] = None,
+        categories: Optional[List[str]] = None,
+        default_run_build: Optional[str] = None,
+        default_run_memory_mbytes: Optional[int] = None,
+        default_run_timeout_secs: Optional[int] = None,
+        example_run_input_body: Optional[Any] = None,
+        example_run_input_content_type: Optional[str] = None,
+    ) -> Dict:
+        """Update the actor with the specified fields.
+
+        https://docs.apify.com/api/v2#/reference/actors/actor-object/update-actor
+
+        Args:
+            name (str, optional): The name of the actor
+            title (str, optional): The title of the actor (human-readable)
+            description (str, optional): The description for the actor
+            seo_title (str, optional): The title of the actor optimized for search engines
+            seo_description (str, optional): The description of the actor optimized for search engines
+            versions (list of dict, optional): The list of actor versions
+            restart_on_error (bool, optional): If true, the main actor run process will be restarted whenever it exits with a non-zero status code.
+            is_public (bool, optional): Whether the actor is public.
+            is_deprecated (bool, optional): Whether the actor is deprecated.
+            is_anonymously_runnable (bool, optional): Whether the actor is anonymously runnable.
+            categories (list of str, optional): The categories to which the actor belongs to.
+            default_run_build (str, optional): Tag or number of the build that you want to run by default.
+            default_run_memory_mbytes (int, optional): Default amount of memory allocated for the runs of this actor, in megabytes.
+            default_run_timeout_secs (int, optional): Default timeout for the runs of this actor in seconds.
+            example_run_input_body (Any, optional): Input to be prefilled as default input to new users of this actor.
+            example_run_input_content_type (str, optional): The content type of the example run input.
+
+        Returns:
+            dict: The updated actor
+        """
+        actor_fields: Dict[str, Any] = {}
+        if name is not None:
```

Comment:
I was thinking about writing some function for this, but in the end I decided to keep things simple and just do it explicitly like this. I'm not really a fan of extracting all the function arguments using `locals()` and then passing them on, it feels kinda hacky and prone to breaking (plus apparently `locals()` doesn't have a defined semantic, and it behaves differently when a debugger is attached, see https://www.python.org/dev/peps/pep-0558/, maybe we should revisit its usage in our code). The other thing is that I can't use `_filter_out_none_values_recursively()` here, because it doesn't filter out nested empty dictionaries (and I didn't want to change it's behavior, since in other places we actually want to keep those empty dictionaries). So again, I think writing these assignments expicitly is ultimately better, because it will lead to less errors when updating this code in the future, even at the cost of the code ugliness and duplication.

Reply:
@fnesveda what about the following
```
argument_per_key = {
	'name': name,
	'title': title,
	...
}
for key, argument in argument_per_key.items():
	if argument:
		actor_fields[key] = argument
```
This would put all the boilerplate which is hard to read to `argument_per_key` which would be a simple dictionary

Reply:
But it still wouldn't solve the nested dictionaries, those would still have to be solved separately. I don't think it's that unreadable, it's pretty clear what the code does.